.github/workflows/test.yml

```diff
@@ -145,6 +145,7 @@ jobs:
           - opensearchproject/opensearch:1.3.6
           - opensearchproject/opensearch:2.1.0
           - opensearchproject/opensearch:2.2.1
+          - opensearchproject/opensearch:2.5.0
```

Comment:
Can you remove the elasticsearch-oss?

Reply:
Yes, I'll add a check on `version.distribution == opensearch`